sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -362,7 +362,6 @@ export interface CosmosClientOptions {
     resourceTokens?: {
         [resourcePath: string]: string;
     };
-    // Warning: (ae-forgotten-export) The symbol "TokenProvider" needs to be exported by the entry point index.d.ts
```

Comment:
@bterlson @HarshaNalluru Can these be errors instead of warnings?

Reply:
I think this might work.

Setting `"logLevel": "error"` in the `api-extractor.json` file.

![image](https://user-images.githubusercontent.com/10452642/65794319-b8f66480-e11c-11e9-9d7e-bd462c51f5c4.png)

Link - https://api-extractor.com/pages/commands/config_file/

Reply:
Below is the config that works, but it seems to have issues. Things that are exported are being incorrectly reported as not being exported. Going to leave it off for now.

```
      "ae-forgotten-export": {
        "logLevel": "error",
        "addToApiReportFile": false
      },
```